cmd/karavictl/cmd/inject.go

```diff
@@ -125,10 +136,13 @@ func NewInjectCmd() *cobra.Command {
 		},
 	}
 
+	//karavictl inject --insecure=true --image-addr amaas-eos-mw1.cec.lab.emc.com:5046/sidecar-proxy:aaron --proxy-host lglap155.lss.emc.com --proxy-port powerflex=10000 --proxy-port powermax=102000
+
 	injectCmd.Flags().String("proxy-host", "", "Help message for proxy-host")
 	injectCmd.Flags().String("image-addr", "", "Help message for image-addr")
 	injectCmd.Flags().Bool("insecure", false, "Allow insecure connections from sidecar-proxy to proxy-server (default: false)")
 	injectCmd.Flags().String("root-certificate", "", "The root certificate file used by the proxy server")
+	injectCmd.Flags().StringSlice("proxy-port", []string{}, "proxy start port in the form <storageSystemType>=<startinPort>")
```

Comment:
Typo "startinPort"

Reply:
Should be fixed by:

https://github.com/dell/karavi-authorization/pull/86/commits/4f914dca2ce92655ca2be27ea98aaae295b42962